README.rst

```diff
@@ -66,6 +66,36 @@ Quickstart
 2. Run ``tutor local launch``
 3. You're done!
 
+.. _compatibility_notes_start:
```

Comment:
I'd rather not include these compatibility notes in the README, nor in the documentation frontpage. That information belongs only in the install docs.